index.html

```diff
@@ -1,139 +1,105 @@
+    rel="apple-touch-startup-image" />
+  <link href="/images/ipadair_splash.png"
+    media="(device-width: 820px) and (device-height: 1180px) and (-webkit-device-pixel-ratio: 2)"
+    rel="apple-touch-startup-image" />
+  <link href="/images/ipadmini5_splash.png"
+    media="(device-width: 768px) and (device-height: 1024px) and (-webkit-device-pixel-ratio: 2)"
+    rel="apple-touch-startup-image" />
+  <link href="/images/ipadmini6_splash.png"
+    media="(device-width: 744px) and (device-height: 1133px) and (-webkit-device-pixel-ratio: 2)"
+    rel="apple-touch-startup-image" />
+  <link href="/images/ipadpro105_splash.png"
+    media="(device-width: 834px) and (device-height: 1112px) and (-webkit-device-pixel-ratio: 2)"
+    rel="apple-touch-startup-image" />
+  <link href="/images/ipadpro11_splash.png"
+    media="(device-width: 834px) and (device-height: 1194px) and (-webkit-device-pixel-ratio: 2)"
+    rel="apple-touch-startup-image" />
+  <link href="/images/ipadpro129_splash.png"
+    media="(device-width: 1024px) and (device-height: 1366px) and (-webkit-device-pixel-ratio: 2)"
+    rel="apple-touch-startup-image" />
+  <link rel="icon" type="image/png" sizes="192x192" href="/images/apple-icon.png" />
+  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png" />
+  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png" />
+  <link rel="icon" type="image/png" sizes="70x70" href="/images/ms-icon-70x70.png" />
+  <link rel="icon" type="image/png" sizes="144x144" href="/images/ms-icon-144x144.png" />
+  <link rel="icon" type="image/png" sizes="150x150" href="/images/ms-icon-150x150.png" />
+  <link rel="icon" type="image/png" sizes="310x310" href="/images/ms-icon-310x310.png" />
+
+
+  <!--
       manifest.json provides metadata used when your web app is added to the
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
-    <link rel="manifest" href="./manifest.json" />
-    <link rel="shortcut icon" href="./favicon.ico" />
-    <link rel="stylesheet" href="index.css">
-    <title>Gordon 360</title>
+  <link rel="manifest" href="./manifest.json" />
+  <link rel="shortcut icon" href="./favicon.ico" />
+  <link rel="stylesheet" href="index.css">
+  <title>Gordon 360</title>
 
-    <meta name="description" content="Gordon 360 is the student app for Gordon College." />
-  </head>
+  <meta name="description" content="Gordon 360 is the student app for Gordon College." />
+</head>
 
-  <body>
-    <noscript> You need to enable JavaScript to use Gordon 360. </noscript>
-    <div id="root"><body style="background-image: linear-gradient(to right, #014983 0%, 
+<body>
+  <noscript> You need to enable JavaScript to use Gordon 360. </noscript>
+  <div id="root">
+    <div style="width: 100%; height: 100%; background-image: linear-gradient(to right, #014983 0%, 
```

Comment:
Most "changes" to this file were just autoformatting. The only contentful changes I made were to the element inside the root div. Previously, this was a duplicate `body` tag (that was unclosed) which is invalid html. I made it a `div` instead and added `width: 100%; height: 100%;` to it's inline styles.